portindex2json/portindex2json.tcl

```diff
@@ -64,7 +67,65 @@ proc remove_extra_braces {text} {
     return [regsub -all {[\{\}]} $text ""]
 }
 
-set fd [open [lindex $argv 0] r]
+proc print_usage {} {
+    puts stdout ""
+    puts stdout "Usage:"
+    puts stdout "    /path/to/tclsh portindex2json.tcl \[--info <key>=<value>\] <path/to/PortIndex>"
+    puts stdout "    /path/to/tclsh portindex2json.tcl --help"
+    puts stdout ""
+    puts stdout "Each '--info <key>=<value>' combinaton adds one key-value pair to the JSON output inside the parent key: \"info\""
+    puts stdout "You may provide any number of '--info <key>=<value>' combinations."
+    puts stdout ""
+}
+
+proc parse_options {} {
+    global argc argv fd
```

Comment:
```suggestion
    global argc argv fd json_info
```

Reply:
I forgot to push this change. If `json_info` is missing, the info field doesn't get added.